mc-oblivious-ram/src/evictor/mod.rs

```diff
@@ -0,0 +1,60 @@
+//! Evictor functions for ORAM
+//!
+//! These are intended to be a module containing different eviction strategies
+//! for tree based orams which include path oram and circuit oram. These
+//! strategies will be used for evicting stash elements to the tree oram.
```

Comment:
:upside_down_face: 
Notes:
* "intended to be a module".  Suggesting a more concrete/decisive wording. It will discourage others from thinking they are the exception and dropping miscellaneous behavior here.  I often want to write in a softer tone too as the design is playing out, but rarely does one come back and address this softer tone if this part of the design is deemed good as is.
* oram -> ORAM.  mainly done for consistency the first line of the module uses `ORAM` and it is an acronym for "Oblivious Random Access Machine". I thought it was oblivious random access memory, until reading through the circuit ORAM paper.
```suggestion
//! A module containing different eviction strategies for tree based ORAMs which
//! include path ORAM and circuit ORAM. These strategies will be used for
//! evicting stash elements to the tree ORAM.
```